vmicore/test/lib/plugins/mock_PluginSystem.h

```diff
@@ -6,6 +6,11 @@ namespace VmiCore
     class MockPluginSystem : public IPluginSystem
     {
       public:
+        MOCK_METHOD(std::unique_ptr<IMemoryMapping>,
+                    mapProcessMemoryRegion,
+                    (addr_t, addr_t, std::size_t),
+                    (const override));
+
         MOCK_METHOD(std::unique_ptr<std::vector<uint8_t>>,
```

Comment:
std::size_t

Reply:
The actual problem is that this function should not exist anymore. :+1: Apparently that's not a compile time error for whatever reason...